src/features/klant/bedrijf/service.ts

```diff
@@ -17,38 +17,39 @@ import type {
 } from "./types";
 
 export const bedrijfQuery = <K extends SearchCategories>(
-  query: BedrijfQuery<K>
+  query: BedrijfQuery<K>,
 ) => query;
 
 type KvkPagination = {
   pagina: number;
-  aantal: number;
+  resultatenPerPagina: number;
   totaal: number;
   resultaten: any[];
 };
 
 const parseKvkPagination = async ({
   pagina,
-  aantal,
+  resultatenPerPagina,
   totaal,
   resultaten,
 }: KvkPagination): Promise<Paginated<Bedrijf>> => ({
   page: await Promise.all(resultaten.map(mapHandelsRegister)),
   pageNumber: pagina,
   totalRecords: totaal,
-  pageSize: aantal,
-  totalPages: totaal === 0 ? 0 : Math.ceil(totaal / aantal),
+  pageSize: resultatenPerPagina,
+  totalPages: totaal === 0 ? 0 : Math.ceil(totaal / resultatenPerPagina),
 });
 
 const handelsRegisterBaseUrl = "/api/kvk";
+const zoekenUrl = `${handelsRegisterBaseUrl}/v2/zoeken`;
```

Comment:
er is nu dus een v1 en een v2 url tegelijkertijd in gebruik. dat klopt?
zoeken -> `${handelsRegisterBaseUrl}/v2/zoeken
vestiging -> handelsRegisterBaseUrl + "/v1/vestigingsprofielen/"
misschien iets consistenter opbouwen? bv beiden hier als constanten, maakt het component wat overzichtelijker

Reply:
in overleg aangepast en code voor overzichtelijkheid wat verder uitgesplitst